src/main/java/seedu/address/model/person/Exam.java

```diff
@@ -82,9 +83,16 @@ public static boolean isValidExamName(String name) {
     /**
      * Returns true if a given date is a valid exam date (from today onwards).
      */
-    public static boolean isValidExamDate(LocalDate test) {
+    public static boolean isValidExamDate(LocalDate test) throws IllegalArgumentException {
         LocalDate currentDate = LocalDate.now();
-        return !test.isBefore(currentDate);
+        try {
```

Comment:
i think checking whether test time is before current time is good too

Reply:
Yaa i think it'll be good, but this can't be done, it'll be considered a feature flaw